runtime/gc_api/HeapIteratorAPI.cpp

```diff
@@ -524,6 +525,47 @@ j9mm_iterate_all_ownable_synchronizer_objects(J9VMThread *vmThread, J9PortLibrar
 	return returnCode;
 }
 
+/**
+ * Walk all continuation objects, call user provided function.
+ * @param flags The flags describing the walk (unused currently)
+ * @param func The function to call on each object descriptor.
+ * @param userData Pointer to storage for userData.
+ * @return return 0 on successfully iterating entire list, return user provided function call if it did not return JVMTI_ITERATION_CONTINUE
+ */
+jvmtiIterationControl
+j9mm_iterate_all_continuation_objects(J9VMThread *vmThread, J9PortLibrary *portLibrary, UDATA flags, jvmtiIterationControl (*func)(J9VMThread *vmThread, J9MM_IterateObjectDescriptor *object, void *userData), void *userData)
+{
+	J9JavaVM *javaVM = vmThread->javaVM;
+	MM_GCExtensions *extensions = MM_GCExtensions::getExtensions(javaVM->omrVM);
+	MM_ObjectAccessBarrier *barrier = extensions->accessBarrier;
+	MM_ContinuationObjectList *continuationObjectList = extensions->getContinuationObjectListsExternal(vmThread);
+
+	Assert_MM_true(NULL != continuationObjectList);
```

Comment:
is this really true?

Reply:
yes, there are an array of lists per region, though there could be zero item in the list.